### PR TITLE
add clean support, add timeout

### PR DESCRIPTION
**Pull request by mostlikelee**
#18456

to be paired with https://github.com/fleetdm/fleet/pull/18457

### CODE REVIEW

File: .github/workflows/generate-cve.yml [hunk 1]
```diff
@@ -56,9 +63,7 @@ jobs:
         run: |
           cd fleet
           go mod download
-          export VULNCHECK_API_KEY=${{ secrets.VULNCHECK_API_KEY }}
           go run cmd/cve/generate.go --db_dir ./cvefeed --debug
-          rm ./cvefeed/vulncheck.zip
```
Comment (mostlikelee) [inline .github/workflows/generate-cve.yml:61]: unused.  cleanup is performed in generate.go

File: .github/workflows/generate-cve.yml [hunk 2]
```diff
@@ -56,9 +63,7 @@ jobs:
         run: |
           cd fleet
           go mod download
-          export VULNCHECK_API_KEY=${{ secrets.VULNCHECK_API_KEY }}
```
Comment (mostlikelee) [inline .github/workflows/generate-cve.yml:59]: redundant with the env key above

File: .github/workflows/generate-cve.yml [hunk 3]
```diff
@@ -7,17 +7,24 @@ on:
   schedule:
     - cron: '*/30 * * * *'
   workflow_dispatch:
+    inputs:
+      clean:
+        description: 'Download fresh feeds from NVD'
```
Comment (lucasmrod) [inline .github/workflows/generate-cve.yml:12]: Document that any value other than `false`/empty means a clean run.

File: .github/workflows/generate-cve.yml (hunk 1 above)
Comment (lucasmrod) [inline .github/workflows/generate-cve.yml:61]: Why wasn't it failing (exit code 1 for `rm`)?

**Review by lucasmrod — APPROVED** [review]

File: .github/workflows/generate-cve.yml (hunk 1 above)
Comment (mostlikelee) [reply .github/workflows/generate-cve.yml:61]: good q, it was failing in my personal repo....taking a look

Comment (mostlikelee) [reply .github/workflows/generate-cve.yml:61]: something is weird here, that's not in `main`

Comment (lucasmrod) [reply .github/workflows/generate-cve.yml:61]: Indeed...

Comment (lucasmrod) [reply .github/workflows/generate-cve.yml:61]: Maybe rebase with latest `main` and push force?

Comment (mostlikelee) [reply .github/workflows/generate-cve.yml]: a rebase cleaned that up

**Review by lucasmrod — APPROVED** [review]

**Pull request merged**